lib/hw_unique_key/hw_unique_key_acl.c

```diff
@@ -42,6 +42,12 @@ bool hw_unique_key_is_written(enum hw_unique_key_slot kmu_slot)
 		HUK_PANIC();
 	}
 
+	uint32_t protection = fprotect_is_protected(huk_addr);
```

Comment:
Not relevant to this line the kmu_slot as a variable here is misleading a bit I think. We can name it as key_slot later for better clarification.

Reply:
Created NCSDK-11414 to track this